src/sentry/features/manager.py

```diff
@@ -211,25 +211,28 @@ def has(
         >>> FeatureManager.has('organizations:feature', organization, actor=request.user)
 
         """
-        actor = kwargs.pop("actor", None)
-        feature = self.get(name, *args, **kwargs)
-
-        # Check registered feature handlers
-        rv = self._get_handler(feature, actor)
-        if rv is not None:
-            return rv
+        try:
+            actor = kwargs.pop("actor", None)
+            feature = self.get(name, *args, **kwargs)
 
-        if self._entity_handler and not skip_entity:
-            rv = self._entity_handler.has(feature, actor)
+            # Check registered feature handlers
+            rv = self._get_handler(feature, actor)
             if rv is not None:
                 return rv
 
-        rv = settings.SENTRY_FEATURES.get(feature.name, False)
-        if rv is not None:
-            return rv
+            if self._entity_handler and not skip_entity:
+                rv = self._entity_handler.has(feature, actor)
+                if rv is not None:
+                    return rv
+
+            rv = settings.SENTRY_FEATURES.get(feature.name, False)
+            if rv is not None:
+                return rv
 
-        # Features are by default disabled if no plugin or default enables them
-        return False
+            # Features are by default disabled if no plugin or default enables them
+            return False
+        except Exception:
```

Comment:
It'll be good to send the exception to Sentry.

Reply:
Sending the exception to Sentry caused alerts to trigger on #team-engineering and made me waste half an hour trying to figure out if the API involved is affecting customers or not (see https://sentry.sentry.io/issues/3922419722/).

I've filed [49905](https://github.com/getsentry/sentry/issues/49905) to fix this.

Reply:
I've thought more about this. We should raise an error during tests.